live/root-ALP-PXE/etc/dracut.conf.d/90-brd.conf

```diff
@@ -0,0 +1 @@
+add_drivers+=" brd "
```

Comment:
Just a comment, **ALP** is dead

Reply:
As @imobachgs mentioned above, let's solve that separately later.